examples/usdt_sample/scripts/lat_avg.py

```diff
@@ -12,23 +12,25 @@
     formatter_class=argparse.RawDescriptionHelpFormatter)
 parser.add_argument("-p", "--pid", type=int, help="The id of the process to trace.")
 parser.add_argument("-i", "--interval", type=int, help="The interval in seconds on which to report the latency distribution.")
-parser.add_argument("-c", "--count", type=int, default=16, help="The count of samples over which to calculate the moving average.")
+parser.add_argument("-m", "--maxsamplesize", type=int, default=16, help="The maximum number of samples over which to calculate the moving average.")
```

Comment:
could we keep `-c`, `--count` here, and add `-m`, `--maxsamplesize` as additional aliases?

Reply:
Done, put --count back, removed --maxsamplesize.